app/code/local/Magmodules/Webwinkelconnect/etc/system.xml

```diff
@@ -198,6 +198,28 @@
                         </resend_double>
                     </fields>
                 </invitation>
+                <product_review_invites>
+                    <label>Invitation to write product review</label>
```

Comment:
This label should be translatable. I see `translate="label"` on the other options. But I'm not sure how translation in Magento works and if you need to do something else.